specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -214,42 +309,45 @@
     },
     "ConversationItemBase": {
       "type": "object",
-      "discriminator": "modality",
       "description": "The abstract base for a user input formatted conversation (e.g., Text, Transcript).",
       "properties": {
-        "participantId": {
-          "description": "The participant Id of a ConversationItem",
+        "id": {
+          "description": "The ID of a conversation item.",
           "type": "string"
         },
-        "id": {
-          "description": "The Id of a ConversationItem",
+        "participantId": {
+          "description": "The participant ID of a conversation item.",
           "type": "string"
         },
         "language": {
-          "description": "The input language of a ConversationItem in BCP 47 language representation.",
+          "description": "The override language of a conversation item in BCP 47 language representation.",
           "type": "string"
         },
-        "modality": {
-          "description": "The modality (format) of ConversationItem (e.g., Text, Transcript)",
+        "inputModality": {
+          "$ref": "#/definitions/InputModality"
+        },
+        "role": {
           "type": "string",
+          "description": "The role of the participant.",
           "enum": [
-            "text"
+            "agent",
+            "customer",
+            "generic"
           ],
           "x-ms-enum": {
-            "name": "modality",
+            "name": "role",
             "modelAsString": true
           }
         }
       },
       "required": [
         "participantId",
-        "id",
-        "modality"
+        "id"
       ]
     },
     "TextConversationItem": {
       "description": "The text modality of an input conversation.",
-      "x-ms-discriminator-value": "text",
+      "x-ms-discriminator-value": "Text",
```

Comment:
The property is `text`, though. Maybe search-and-(over)replace?

Reply:
Actually reverting back to "text" since CustomConv is using this as "lowercase"